src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -295,6 +302,12 @@ public long Length
         /// </summary>
         public string Name => ParseFileName(FullName, _versionMadeByPlatform);
 
+        internal ZipArchive.ChangeState Changed { get; private set; }
```

Comment:
nit: "Changed" -> "Changes"?

"Changed" sounds like a bool, while "Changes" sounds like the flags it is.